src/components/fetchhelper.ts

```diff
@@ -73,13 +73,13 @@ export function paramsToString(params) {
     return values.join('&');
 }
 
-export function ajax(request) {
+export function ajax(request: any): Promise<any> {
```

Comment:
Same as above

Reply:
Here the request is a custom type that needs its own definition. The response will be interpreted to various objects througout the code, so it should really be templated somehow.